diff --git a/go/tools/bazel_testing/bazel_testing.go b/go/tools/bazel_testing/bazel_testing.go
index 4d1ea2f8..0bad0399 100644
--- a/go/tools/bazel_testing/bazel_testing.go
+++ b/go/tools/bazel_testing/bazel_testing.go
@@ -441,6 +441,7 @@ func loadWorkspaceName(workspacePath string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	workspaceData = bytes.ReplaceAll(workspaceData, []byte("\r\n"), []byte("\n"))
 	nameRe := regexp.MustCompile(`(?m)^workspace\(\s*name\s*=\s*("[^"]*"|'[^']*')\s*,?\s*\)$`)
 	match := nameRe.FindSubmatchIndex(workspaceData)
 	if match == nil {
